src/InlineHelp/InlineHelp.js

```diff
@@ -20,22 +21,18 @@ const InlineHelp = React.forwardRef(({ text, placement, className, contentClassN
         className
     );
 
-    const inlineHelpContentClasses = classnames(
-        'fd-inline-help__content',
-        {
-            [`fd-inline-help__content--${placement}`]: !!placement
-        },
-        contentClassName
-    );
-
     return (
-        <span className={inlineHelpClasses} ref={ref}>
-            <span
-                {...props}
-                className={inlineHelpContentClasses}>
-                {text}
-            </span>
-        </span>
+        <Popover
+            body={
+                <div {...props}
+                    className='fd-inline-help__content fd-no-border' >
+                    {text}
+                </div>
+            }
+            control={<span><span
```

Comment:
Can we get rid of the extra span?

Reply:
This was added since the ref was returning null when the ref was added to the parent element. Tried to add react fragment(<></>) but it was also failing.

Reply:
It seems like Popover might be overwriting the ref prop [here](https://github.com/SAP/fundamental-react/blob/InlineHelp-rebuild-with-Popover-component/src/Popover/Popover.js#L136)

It could do something like this there
```jsx
            ref: (c) => {
                this.controlRef = findDOMNode(c);
                if (control.props.ref) {
                   control.props.ref(c);
                }
            }
```

Reply:
related conversation: https://github.com/SAP/fundamental-react/pull/1082#discussion_r434924795